tests/attacks/evasion/test_shadow_attack.py

```diff
@@ -58,7 +58,7 @@ def test_generate(art_warning, fix_get_mnist_subset, image_dl_estimator_for_atta
 
         x_train_mnist_adv = attack.generate(x=x_train_mnist[0:1], y=y_train_mnist[0:1])
 
-        assert np.max(np.abs(x_train_mnist_adv - x_train_mnist[0:1])) == pytest.approx(0.34966960549354553, 0.06)
+        assert np.max(np.abs(x_train_mnist_adv - x_train_mnist[0:1])) == pytest.approx(0.38116083, 0.06)
```

Comment:
Why did this expectation value change?

Reply:
oh ye that was a nasty one ! :D so while I was trying to bring all the tests that were not in the main loop, this test kept failing and I couldn't understand why because I hadn't changed anything at all in there. Then I discovered that that test was never run because it had the `@pytest.mark.only_with_platform("pytorch")` marker but was in a loop which never ran the tests with pytorch which is why it was not failing before and failing for pytorch in the main loop. So since I haven't made any changes to the models in this PR I it was ok if I updated the value that the pytorch model actually outputs to make sure that test can now be run as part of the main loop. I'm not sure if this makes sense?